simpleui-server/src/simpleui-server.ts

```diff
@@ -31,11 +29,7 @@ export class SimpleUIServer {
     static bin_dir = "";
     static newMockDataURL: any = ""; // allows us to modify the mock data requests via mock cmd requests
     static webPortString = "";
-    static SIGNALS = {
```

Comment:
Are we still gracefully exiting on SIGHUP, SIGINT, and SIGTERM?
Lines 450-457 below are deleted in new version.  Where did signal handling move to?

Reply:
In the [Dockerfile](https://github.com/greg-ses/simpleui/blob/zmq-resource-leak/Dockerfile#L85), we execute `run.sh` with an init system called [Tini](https://github.com/krallin/tini) which takes care of most of the graceful shutdowns for us which is nice. I have specific event handlers for the [zmq sockets](https://github.com/greg-ses/simpleui/blob/zmq-resource-leak/simpleui-server/src/sui_zmq.ts#L153), which closes the sockets before the process shuts down. But now that I think of it, do those sockets need to be closed before we shut down? Could I replace that whole section with a hard exit `process.exit()`?